### PR TITLE
Issue 2050

### PR DESCRIPTION
**Pull request by cedricguindon**

### CODE REVIEW

File: armTemplates/azuredeploy.json [hunk 1]
```diff
@@ -1,1316 +1,1566 @@
...
+			}
+		},
+		"databaseApplicationPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The Matomo application database account password."
+			}
+		},
+		"databaseApplicationUsername": {
+			"type": "string",
```
Comment (PatrickMessierGoC) [inline armTemplates/azuredeploy.json:74]: securestring

File: armTemplates/azuredeploy.json [hunk 2]
```diff
@@ -1,1316 +1,1566 @@
...
+			}
+		},
+		"webServerAdminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The Web server administrator's password."
+			}
+		},
+		"webServerAdminUsername": {
+			"type": "string",
```
Comment (PatrickMessierGoC) [inline armTemplates/azuredeploy.json:288]: securestring

File: armTemplates/azuredeploy.json [hunk 3]
```diff
@@ -1,1316 +1,1566 @@
...
+					},
+					{
+						"name": "[variables('applicationSubnetName')]",
+						"properties": {
+							"addressPrefix": "[variables('applicationSubnetAddressPrefix')]",
+							"networkSecurityGroup": {
+								"id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('applicationNetworkSecurityGroupName'))]"
+							},
+							"serviceEndpoints": [
+								{
```
Comment (PatrickMessierGoC) [inline armTemplates/azuredeploy.json:466]: À supprimer

File: armTemplates/azuredeploy.json [hunk 4]
```diff
@@ -1,1316 +1,1566 @@
...
+				]
+			}
+		},
+		{
+			"type": "Microsoft.Network/privateDnsZones",
+			"apiVersion": "2018-09-01",
+			"name": "[variables('privateDnsZoneName')]",
+			"location": "global",
+			"properties": {
+				"maxNumberOfRecordSets": 25000,
```
Comment (PatrickMessierGoC) [inline armTemplates/azuredeploy.json:641]: à supprimer. 
Ref.: https://learn.microsoft.com/en-us/azure/templates/microsoft.network/2018-09-01/privatednszones?pivots=deployment-language-arm-template#privatezoneproperties-1

**Review by PatrickMessierGoC — CHANGES_REQUESTED** [review]

File: armTemplates/azuredeploy.json (hunk 2 above)
Comment (cedricguindon) [reply armTemplates/azuredeploy.json:288]: done

File: armTemplates/azuredeploy.json (hunk 1 above)
Comment (cedricguindon) [reply armTemplates/azuredeploy.json:74]: done

File: armTemplates/azuredeploy.json [hunk 5]
```diff
@@ -1,1316 +1,1566 @@
...
+			}
+		},
+		"databaseAdminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The password to use in order to manage the database server."
+			}
+		},
+		"databaseAdminUsername": {
+			"type": "string",
```
[the remark this answers is not in the document]
Comment (cedricguindon) [reply armTemplates/azuredeploy.json:62]: done

File: README.md [hunk 6]
```diff
@@ -13,7 +13,7 @@ This project deploys the following Azure resources:
 
 And installs the following software (up to their latest available patch level for the linux distro) on the virtual machine:
 - Ubuntu 18.04
-- Mysql client 5.7
+- Mysql client 8.0
```
[the remark this answers is not in the document]
Comment (cedricguindon) [reply README.md:16]: done

File: taskGroups/DeployMatomoSingleNode.json [hunk 7]
```diff
@@ -21,7 +21,7 @@
...
...SizeGB $(dataDiskSizeGB) -deniedIpAddressList \"$(deniedIpAddressList)\" -environmentName \"$(environmentName)\" -gatewaySslCertificateUri \"$(gatewaySslCertificateUri)\" -gatewayUserAssignedIdentity $(gatewayUserAssignedIdentity) -genericShellScriptsPublicRawFilesUrl \"$(genericShellScriptsPublicRawFilesUrl)\" -geoRedundantBackup \"$(geoRedundantBackup)\" -highAvailabilityMode \"$(highAvailabilityMode)\" -location \"$(location)\" -projectPublicRawFilesBaseUrl \"$(projectPublicRawFilesBaseUrl)\" -projectShortName \"$(projectShortName)\" -resourceTags $(resourceTags) -smtpServerFqdn \"$(smtpServerFqdn)\" -smtpServerPort \"$(smtpServerPort)\" -smtpServerPrivateIp \"$(smtpServerPrivateIp)\" -standbyAvailabilityZone \"$(standbyAvailabilityZone)\" -virtualMachineTags $(virtualMachineTags) -virtualNetworkAddressBase \"$(virtualNetworkAddressBase)\" -webServerAdminEmail \"$(webServerAdminEmail)\" -webServerAdminPassword \"$(webServerAdminPassword)\" -webServerAdminUsername \"$(webServerAdminUsername)\" -webServerBackendMaxExecutionTime $(webServerBackendMaxExecutionTime) -webServerRebootUtcTime \"$(webServerRebootUtcTime)\" -webServerVirtualMachineSize \"$(webServerVirtualMachineSize)\"",
```
[the remark this answers is not in the document]
Comment (cedricguindon) [reply taskGroups/DeployMatomoSingleNode.json:24]: done

File: armTemplates/azuredeploy.json (hunk 4 above)
Comment (cedricguindon) [reply armTemplates/azuredeploy.json:641]: done

**Review by PatrickMessierGoC — APPROVED** [review]

**Pull request merged**